packages/debug/src/browser/style/start.svg

```diff
@@ -1,4 +1,6 @@
 <!--Copyright (c) Microsoft Corporation. All rights reserved.-->
-<!--Copyright (C) 2018 TypeFox and others.-->
+<!--Copyright (C) 2020 TypeFox and others.-->
```

Comment:
I think this is the dark/inverse version of this icon. Replace with light version?